ieeeuoft/hackathon-template
# Added hardware count selector to inventory page

## Overview

- Resolves [IEEE-90](https://ieeeuoft.atlassian.net/browse/IEEE-90)
- Added a hardware count selector in hardware slice
- Displays SHOWING {number of hardware items queried} OF {total hardware items} ITEMS
- Displays total number of items beside search bar
- Displays "Load More" button if the amount of hardware items retrieved is less than the total number of items

## Unit Tests Created

- Default hardware count
- hardware count with filtering by category


## Steps to QA

- Use the inventory page normally (filter by any category, try ordering too) and see if the hardware counts change in the places described above
